tests/conftest.py

```diff
@@ -48,3 +49,30 @@ def undo():
             monkeypatch.setattr('click.echo', old_echo)
 
     return Result
+
+
+@pytest.fixture
+def sleep_time(tmpdir):
+    """
+    Returns how long we need to sleep for the filesystem's mtime precision to
+    pick up differences.
```

Comment:
I'm not entirely sure if [`os.stat_float_times`](https://docs.python.org/3/library/os.html#os.stat_float_times) already does this. This one is certainly more robust regarding misbehaving filesystems on `/tmp` (which a global setting can't detect).